docs/administrator.md

```diff
@@ -804,6 +804,52 @@ pod_environment_configmap: "postgres-operator-system/pod-env-overrides"
 ...
 ```
 
+### Azure Storage Account Setup
+
+To configure phyiscal backups and WAL to be sent to an azure storage account perform the
+following steps.
+
+1. Generate the K8s secret resource that will contain your service account's
+credentials. It's highly recommended to use a service account and limit its
```

Comment:
Can you actually use a service principal for this? It seems like you can only use an access token or SAS token. I don't see any WAL-G configuration to provide access via a SP.

Reply:
I've removed the reference to service-account.